networks/example/live-with-isp/configs/as2border1.cfg

```diff
@@ -129,13 +129,16 @@ no ip http server
 no ip http secure-server
 !
 ip access-list extended INSIDE_TO_AS1
- permit ip 2.0.0.0 0.255.255.255 1.0.0.0 0.255.255.255
+ permit ip 2.0.0.0 0.255.255.255 any
  permit ip 10.12.11.2 0.0.0.0 10.12.11.1 0.0.0.0 
+ permit tcp any eq 179 any
```

Comment:
syntax is incorrect... should be `permit tcp any any eq 179` like the one below

Reply:
The line as written, only permits traffic outbound where TCP 179 is the source port. So the remote router must initiate the TCP connection for BGP session to be established.